src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -258,7 +273,7 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None):
             # XXX: Carefuly, this variable will not exist in `seq2seq` setting.
             # Currently chunking is not possible at this level for `seq2seq` so
             # it's ok.
-            align_to = self.model.config.inputs_to_logits_ratio
+            align_to = getattr(self.model.config, "inputs_to_logits_ratio", 1)
```

Comment:
@Narsil ok for you or should we add `1` to the model's config?

Reply:
I personally wouldn't add `1` to the config because we actually don't know what the "inputs to logits ratio" is. In other models like wav2vec, `inputs_to_logits_ratio` is a property that calculates the product of the strides of the convolutional layers, but we can't do that with `M-CTC-T`. As far as I can tell, it needs to be calculated dynamically.

This is assigning `align_to`, which is used on the subsequent lines to try and make sure that the chunk length and stride lengths are even multiples of that ratio. But in the case we don't know what that ratio is yet, setting `align_to = 1` simply aligns the chunk length and stride lengths to the nearest frame instead.

Reply:
It's mostly used for getting `timestamps` correct, and this PR makes them pretty off.